content/300-guides/050-database/100-developing-with-prisma-migrate/165-enable-native-database-functions.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Enable extensions for native database functions
-metaDecription: How to enable native database functions for projects that use Prisma Migrate.
+metaDescription: How to enable native database functions for projects that use Prisma Migrate.
```

Comment:
ℹ️ Wanted to call your attention to this because I changed the Front Matter property name, assuming the thing referencing it uses the correct name. So code may need to change elsewhere to allow fixing this spelling issue.